share/scripts/tcl_bridge.tcl

```diff
@@ -0,0 +1,93 @@
+# tcl_bridge v1.0 (c) 2025 Pedro de Medeiros
```

Comment:
To make this an _official_ openMSX Tcl script we should:
* Rename the script from `tcl_bridge.tcl` to `_tcl_bridge.tcl`.
* Register the script in `lazy.tcl`.

To improve openMSX startup time, we don't automatically execute tcl scripts in `.openMSX/share/scripts` that start with an `_` character. Instead we only do that on-demand (lazy) and to do that we need a little extra information that's provided via `lazy.tcl`.